pyproject.toml

```diff
@@ -21,6 +21,8 @@ Changelog = "https://github.com/hacf-fr/renault-api/releases"
 [tool.poetry.dependencies]
 python = "^3.7.1"
 click = "^7.0"
+aiohttp = "^3.7.2"
+typing-extensions = "^3.7.4"
```

Comment:
It is a dependency of aiohttp, and it was causing issues with mypy.
It seems to be sufficient inside dev dependencies to I will move it there now.

Reply:
Strange sub dependencies should be managed installed by poetry. I will investigate. Can you confirm the version of poetry you have installed ?

Reply:
Poetry version 1.1.4 on my machine, but the problems occured inside

Reply:
Poetry version 1.1.x is new and has regression on export features (see [python-poetry/poetry#3189](https://github.com/python-poetry/poetry/issues/3189)).

When you use it locally poetry generate bad dependencies resolution in the `poetry.lock` file. This file is commited in git and reuse by the CI whatever poetry version we have pinned in the CI. So this could lead to strange results.

Do you mind trying to install locally poetry 1.0.10, remove the `typing-extensions` in `pyproject.toml`  run the command `poetry lock` and check if you still have the issue with mypy ?